### ISSUE TITLE
Missing the Fit::FillMax

### ISSUE THREAD

**Issue by JelmerIJff** [spatie/laravel-medialibrary#3612]
I'm missing the FillMax manipulation from the previous version. 

I've tried Fit::Contain but then the image doesn't get my desired width and height. And with Fit::Fill the image stays really small with a lot of padding around it. FillMax is the perfect solution because that should respect the aspect ratio within the given desired width and height, and at the same time should upscale to fit the given width and height.

**Comment by JelmerIJff** [comment]
No, i've already tried that but then you get a small image with a lot of padding around it. FillMax is supposed to fill it right to the edges right while maintaining the aspect ratio?
![Team Visma Shirt](https://github.com/spatie/laravel-medialibrary/assets/19327303/fd22164d-799a-4947-9250-606aa3037bf8)

**Comment by timvandijck** [comment]
Ok, tested it with some different images and sizes, and I think I see what you mean now.

I implemented a quick implementation in Imagick, but I'll only get around GD next week I'm afraid: https://github.com/spatie/image/pull/258

**Comment by timvandijck** [comment]
Pushed a new release of spatie/image that adds FillMax as a fit operation: https://github.com/spatie/image/pull/258
